test/RoaveTest/SecurityAdvisories/AdvisorySources/GetAdvisoriesFromFriendsOfPhpTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/*
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * This software consists of voluntary contributions made by many individuals
+ * and is licensed under the MIT license.
+ */
+
+declare(strict_types=1);
+
+namespace RoaveTest\SecurityAdvisories\AdvisorySources;
+
+use PHPUnit\Framework\TestCase;
+use Roave\SecurityAdvisories\Advisory;
+use Roave\SecurityAdvisories\AdvisorySources\GetAdvisoriesFromFriendsOfPhp;
+
+class GetAdvisoriesFromFriendsOfPhpTest extends TestCase
+{
+    public function testThatAdvisoriesAreBuiltFromYamlFiles(): void
+    {
+        $advisories = new GetAdvisoriesFromFriendsOfPhp(
+            getcwd().'/test/RoaveTest/SecurityAdvisories/AdvisorySources/security-advisories'
```

Comment:
Should use `__DIR__`, since `getcwd()` can change depending on how `vendor/bin/phpunit` is being run

Reply:
thanks!